packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -33,16 +32,16 @@ The MyAccount API has the following operations:
 * [Verify My Email](#verify-my-email)
 * [Poll My Email Challenge](#poll-my-email-challenge)
 * [Delete My Email](#delete-my-email)
-* [Add My Phone](#add-my-phone)
 * [Get My Phone](#get-my-phone)
 * [Get My Phones](#get-my-phones)
```

Comment:
nit: `Get My Emails` goes before `Get My Email`, should we make it consistent by putting `Get My Phones` before `Get My Phone`? Or update the order for email.

Reply:
done reordering `Get My Phones` and `Get My Phone`